real_intent/deliver/events/main.py

```
@@ -0,0 +1,513 @@
+            response = self.anthropic_client.completions.create(
+            model="claude-2.1",  
+            prompt=system + "\n\nHuman:" + user + "\n\nAssistant:", 
+            max_tokens_to_sample=500,
+            temperature=0.5,
+        )
+
+            
+            return response.completion
+
+        except Exception as e:
+            raise Exception(f"Failed to generate summary: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)